src/main/java/com/example/ItransitionConstants.java

```diff
@@ -0,0 +1,10 @@
+package com.example;
+
```

Comment:
JD? and what with name of the class?

Reply:
@Vlada0612 Following your advice, the class has been renamed. added java dock.